src/sheldon_dictionary.erl

```diff
@@ -168,3 +168,30 @@ typo_model(Word) ->
                 end,
                 [Word],
                 [lists:split(I, Word) || I <- lists:seq(0, length(Word))]).
+
+-spec generate_dictionary(term()) -> binary().
+generate_dictionary(Source) ->
+    DefaultDictionary = application:get_env(sheldon, default_dictionary, undefined),
```

Comment:
```suggestion
    DefaultDictionary = application:get_env(sheldon, default_dictionary, Source),
```
I guess this change ☝🏻 would simplify the `case` statement below.

Reply:
Nice! Done.